resources/views/dashboard/admin/roster/edit.blade.php

```diff
@@ -7,452 +7,341 @@
-                                2 => 'Major/ DEL & GMD',
-                                3 => 'Minor Local',
-                                4 => 'Major Local',
-                                5 => 'Minor Approach',
-                                6 => 'Major Approach',
-                                7 => 'Center'
-                            ], $user->max, ['class' => 'form-control', 'disabled']) !!}
-                        </div>
-                    @endif
-                @endif
+                </div>
             </div>
-        </div>
-        @if(Auth::user()->isAbleTo('roster') || Auth::user()->isAbleTo('train'))
             <div class="form-group">
                 <div class="row">
                     <div class="col-sm-6">
-						{!! Form::label('twr_solo_fields', 'Class D Tower Solo Certifications') !!}
-						{!! Form::text('twr_solo_fields', $user->twr_solo_fields, ['class' => 'form-control','maxlength' => 255]) !!}
+                        {!! Form::label('twr_solo_fields', 'Unrestricted Solo Certifications (list facility IDs)') !!}
+                        {!! Form::text('twr_solo_fields', $user->twr_solo_fields, ['class' => 'form-control','maxlength' => 255, $train_disable]) !!}
                     </div>
                     <div class="col-sm-6">
-                    {!! Form::label('twr_solo_expires', 'Tower Solo Expiration Date', ['class' => 'form-label']) !!}
-                    <div class="input-group date" id="datetimepicker1" data-target-input="nearest">
-                        {!! Form::text('twr_solo_expires', $user->twr_solo_expires, ['placeholder' => 'MM/DD/YYYY', 'class' => 'form-control datetimepicker-input', 'data-target' => '#datetimepicker1']) !!}
-                        <div class="input-group-append" data-target="#datetimepicker1" data-toggle="datetimepicker">
-                            <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+                        {!! Form::label('twr_solo_expires', 'Solo Expiration Date', ['class' => 'form-label']) !!}
+                        <div class="input-group date" id="datetimepicker1" data-target-input="nearest">
+                            {!! Form::text('twr_solo_expires', $user->twr_solo_expires, ['placeholder' => 'MM/DD/YYYY', 'class' => 'form-control datetimepicker-input', 'data-target' => '#datetimepicker1', $train_disable]) !!}
+                            <div class="input-group-append" data-target="#datetimepicker1" data-toggle="datetimepicker">
+                                <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+                            </div>
                         </div>
                     </div>
-                    </div>
                 </div>
-            </div>		
-        <hr>
-        @endif
-        @if(Auth::user()->isAbleTo('roster'))
+            </div>
+            <hr>
+            <h6><i class="fas fa-level-up-alt"></i>&nbsp;Tier 1 Facility Certifications</h6>
```

Comment:
See my other (non-review) comment... this section could benefit from some additional separators or a general moving-around of things. It's a bit hard to digest as a grid of dropdowns... it took me a few minutes to get the hang of how to actually use this interface during QA

Reply:
I've asked @Jcalder13 to take a look at this and provide comments. I saw your suggestion above, but it looks like that is going to result in a large amount of unused whitespace on the view. The current layout was designed to get as much of the form onto a single scrollable-frame as possible.